jetson/percep/main.cpp

```diff
@@ -256,18 +252,18 @@ int main() {
     lcm_.publish("/target_list", &arTagsMessage);
     lcm_.publish("/obstacle", &obstacleMessage);
 
-    #if PERCEPTION_DEBUG
-      imshow("depth", depth_img);
-      waitKey(FRAME_WAITKEY);
+    #if PERCEPTION_DEBUG && AR_DETECTION
+      imshow("depth", src);
+      waitKey(1);
+      auto start = std::chrono::high_resolution_clock::now();
+      std::this_thread::sleep_for(0.2s);
+      auto end = std::chrono::high_resolution_clock::now();    
```

Comment:
Are the start and end variables here just for testing? Have you tested that this method of waiting works?

Reply:
I have tested this and it does work. I accidentally included the vars. I shall remove in future

Reply:
If you weren't pressed for time I would have you fix this now. Make an issue for it